src/utils/workspaceUtils.ts

```diff
@@ -67,6 +69,38 @@ export class WorkspaceUtils {
             return resolve(staticResourcesPath);
         });
     }
+
+    static getPackageJson(): object {
+        return JSON.parse(
+            fs.readFileSync(
+                path.join(this.getWorkspaceDir(), PACKAGE_JSON),
+                'utf8'
+            )
+        );
+    }
+
+    static setPackageJson(packageJson: object) {
+        fs.writeFileSync(
+            path.join(this.getWorkspaceDir(), PACKAGE_JSON),
+            JSON.stringify(packageJson, null, TAB_SPACES)
+        );
+    }
+
+    static packageJsonExists(): boolean {
+        return fs.existsSync(path.join(this.getWorkspaceDir(), PACKAGE_JSON));
+    }
+
+    static lwcFolderExists(): boolean {
+        return fs.existsSync(
+            path.join(this.getWorkspaceDir(), WorkspaceUtils.LWC_PATH)
+        );
+    }
+
+    static isSfdxProjectOpened(): boolean {
```

Comment:
This is not correct. You should also add [this line](https://git.soma.salesforce.com/MobilePlatform/salesforcedx-vscode-mobile/blob/browser-container/src/extension.ts#L58) from the code that I shared with you.

Basically `isSfdxProjectOpened()` should never throw and instead should return either `true` or `false` but as you have defined it right now, `getWorkspaceDir()` can throw. So either at the line above or wrap the whole thing in `try-catch`

```
static isSfdxProjectOpened(): boolean {
    try {
        return fs.existsSync(
            path.join(this.getWorkspaceDir(), SFDX_PROJECT_FILE)
        );
    } catch {
        return false;
    }
}
```